packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -101,6 +101,12 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
   menuIconProps?: IIconProps;
 
   /**
+   * If set to true, there will be no separate menu icon added to this control.
+   * This should only be set when the icon for the button is able to communicate that it invokes a menu.
+   */
+  hideMenuIcon?: boolean;
```

Comment:
I guess we could also do it via styling? If there are no styles then don't render the <icon>. Thoughts?

Reply:
Doesn't this do the same:

```tsx
onRenderMenuIcon={ () => false }
```

Reply:
Yeah it would work, I can just do that instead.

Reply:
yeah, i think that's how i've done it in the past. just render nothing.